tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/logging/LogAPITestCase.java

```diff
@@ -198,7 +198,7 @@ public void testTraceLevel() throws BallerinaTestException {
         validateLog(logLines[9], "TRACE", "[]", traceLog);
     }
 
-    @Test(description = "Tests log functionality when all log levels are turned on")
+    @Test(description = "Tests log functionality when all log levels are turned on", enabled = false)
```

Comment:
A different test seems to be failing now

`org.ballerinalang.test.logging.LogAPITestCase > testBasicLogFunctionality`

We may have to disable more than just this.

Reply:
If that is the case shall we disable all the test cases in LogAPITestCase class?

Reply:
Disabled all the test cases in `LogAPITestCase` class.

Reply:
@MadhukaHarith92, please note.

Reply:
Ack.